R/plumber.R

```diff
@@ -609,7 +609,8 @@ Plumber <- R6Class(
               private$default_parsers
             }
           req$argsPath <- h$getPathParams(path)
-          req$argsPostBody <- postbody_parser(req, parsers)
+          # `req_body_parser()` will also set `req$body` with the untouched body value
+          req$argsBody <- req_body_parser(req, parsers)
```

Comment:
Remove the side-effect of this function (return body) and do the arg parsing in another function.